spec/system/support/prior_authority/step_helpers.rb

```diff
@@ -2,6 +2,7 @@
 
 module PriorAuthority
   module StepHelpers
+    # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
(For comparison, I'd be more inclined to let this one slide. In general was want to keep cyclomatic complexity down. In this _specific_ case, unusually, we're ok with it, because while it might be "cyclomatically" complex, it's actually very simple, and very _obviously_ simple.)